src/apps/relay/dtls_listener.c

```diff
@@ -390,15 +395,20 @@ static int handle_udp_packet(dtls_listener_relay_server_type *server,
 			{
 				uint8_t saddr[129];
 				uint8_t rsaddr[129];
-				long thrid = (long) pthread_self();
 				addr_to_string(get_local_addr_from_ioa_socket(chs),saddr);
 				addr_to_string(get_remote_addr_from_ioa_socket(chs),rsaddr);
+				long thrid = 0;
+#ifdef WINDOWS
+				thrid = GetCurrentThreadId();
```

Comment:
Better abstract it into a function that internally has ifdef (Windows)

Otherwise this ifdef is copied over multiple times in code